Localization/Translations/Options/Tracker.lua

```diff
@@ -347,8 +347,8 @@ local trackerOptionsLocales = {
         ["koKR"] = false,
         ["esMX"] = "Cuando esto está marcado, el Rastreador se minimizará automáticamente al entrar a un calabozo.",
         ["enUS"] = true,
-        ["zhCN"] = "选中此项后，进入地下城时跟踪器将自动最小化。",
-        ["zhTW"] = false,
+        ["zhCN"] = false,
```

Comment:
Same as above, why removing it?

Reply:
Same as above. The translation is right. My mistake.